src/en/reference-release-notes.md

```diff
@@ -15,6 +15,167 @@ CARRIAGE RETURN; LISTS ARE TOO HEAVY-HANDED IMO
+  communicate with controllers. Setting a value for this item limits the IP
+  addresses of controller API endpoints in agent config, to those in the space.
+  If the value is misconfigured so as to expose no addresses to agents, then a
+  fallback to all available addresses results. Juju client communication with
+  controllers is unaffected by this value.
+  
+  'juju-ha-space' is the name of the network space used for MongoDB replica-set
+  communication in high availability (HA) setups. This replaces the previously
+  auto-detected space used for such communication. When enabling HA, this value
+  must be set where member machines in a HA set have more than one IP address
+  available for MongoDB use, otherwise an error will be reported. Existing HA
+  replica sets with multiple available addresses will report a warning instead of
+  an error provided the members and addresses remain unchanged.
+  
+  Using either of these options during `bootstrap` or `enable-ha` effectively
+  adds constraints to machine provisioning. The commands will fail with an error
+  if such constraints can not be satisfied.
+  
+  **Cloud credential changes**  
+  Cloud credentials are used by models to authenticate communications with the
+  underlying provider as well as to perform authorised operations on this
+  provider. 
+  
+  Juju has always dealt with both cloud credentials stored locally on a user’s
+  client machine as well as the cloud credentials stored remotely on a
+  bootstrapped Juju controller. The distinction has not been made clear
+  previously and this release addresses these ambiguities.
+  
+  	  juju show-model ...
+  
+  Basic cloud credential information such as its name and owner have been added
+  to the command output. 
+  
+  	  juju show-credential ...
+  
+  This is a new command that shows a logged on user their remotely stored cloud
+  credentials along with models that use them.
+  
+  See command help for more information.
+  
+  ## Fixes
+  
+  For a list of all bugs fixed in this release, see
+  the [2.4-beta1 milestone](https://launchpad.net/juju/+milestone/2.4-beta1).
+  
+  Some important fixes/improvements include:
+  
+  [LP 1755141](https://bugs.launchpad.net/bugs/1755141) add support for --all  
+  [LP 1762979](https://bugs.launchpad.net/bugs/1762979) --no-retry behaviour is inverted  
+  [LP 1753593](https://bugs.launchpad.net/bugs/1753593) support for st1 and sc1 volume types on AWS  
+  [LP 1754735](https://bugs.launchpad.net/bugs/1754735) support for new AWS instance types
+  
+  If you were affected by any of the bugs fixed in this release, your feedback
+  is appreciated. Please contact the Juju team using the communication channels
+  specified in the feedback section.
+  
+  ## Get Juju.
+  
+  The only way to get the development version of Juju is by using the `--edge`
+  channel in the `snap` package:
+  
+  	  sudo snap install juju --edge --classic
+
+  To move to the stable version:
```

Comment:
I would say, "You can revert your Juju client to the latest stable version with: "

Reply:
Doesn't "revert" imply going back to what already was?

Reply:
the point of putting that in there is to tell people how to get back to the stable version, i guess assuming that they had a stable version and then used tohe above instruction to try the new beta.
Word it however you like. The current wording isn't clear and assumes that readers know the latest stable version is something different